build.sh

```diff
@@ -7,14 +7,27 @@ set -e
 DIR=$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)
 cd $DIR
 
+x=0
+for arg in "$@"; do
+	let x=x+1
+	if test "$arg" = "--use-cache"; then
+		KAPACITOR_USE_BUILD_CACHE=defined
+		set -- "${@:1:$(expr $x - 1)}" "${@:$(expr $x + 1):$#}"
+		break
+	fi
+done
 
 # Build new docker image
 docker build -f Dockerfile_build_ubuntu64 -t influxdata/kapacitor-builder $DIR
+if test -n "$KAPACITOR_USE_BUILD_CACHE"; then
+	docker run --entrypoint=/bin/true --name kapacitor-builder-cache influxdata/kapacitor-builder 2>/dev/null || true
+fi
```

Comment:
This line creates a  container  - kapacitor-builder-cache - that owns the /root/go/src volume that acts as the cache of dependencies.